Logic/core/indexer/index.py

```diff
@@ -149,30 +149,63 @@ def check_add_remove_is_correct(self):
             print('Add is incorrect, document')
             return
 
+        if not index_before_add[Indexes.STARS.value].__contains__('tim'):
+            index_before_add[Indexes.STARS.value].setdefault('tim', {})
+

```

Comment:
I think it's better to define a helper function, for example named `check_if_key_exists` and pass all of these (`tim`, `henry`, ...) to it, so it sets the default value of it to `{}`. 
Also, isn't it a better idea to use dictionary's `get` function instead of doing this?

Reply:
In this context(checking the existence of the key without invoking the dictionary using that key), using the ```get``` function doesn't make a difference